lib/bridge/pbn/game_parser.rb

```diff
@@ -0,0 +1,53 @@
+module Bridge
+  module Pbn
+    class GameParser
+      include SingleCharComparisonConstants
+
+      # TODO: enforce 255 char cap on line width
+      def each_subgame(pbn_game_string, &block)
+        @pbn_game_string = pbn_game_string
```

Comment:
Great feedback. those should be in initialize with a builder or abstract factory.  Honestly I think I really failed by specifying master on the fork as the merge source. I haven't worked with forked repos before, so I never really thought about merging properly before. I don't mean to be dirtying up your repo with n00b mistakes and huge change lists. Besides, the right next step at this point seems to be to gemify the domain objects, which I'd be happy to take a crack at in another, simpler pull request, when I find the time.